install_bootstrap

```diff
@@ -47,7 +47,7 @@ function check_for_repo {
 }
 
 function generate_password {
-    tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1
+    tr -cd '[:alnum:]' < /dev/urandom | fold -w32 | head -n1
```

Comment:
There is existing code that uses generated passwords. Were you able to verify that they were not broken by this change?

Probably not, but just making sure.

Reply:
👍